kafka/adminapi.go

```diff
@@ -1841,6 +1913,9 @@ func (a *AdminClient) SetSaslCredentials(username, password string) error {
 
 // Close an AdminClient instance.
 func (a *AdminClient) Close() {
+	if !atomic.CompareAndSwapUint32(&a.isClosed, 0, 1) {
+		return
+	}
 	if a.isDerived {
 		// Derived AdminClient needs no cleanup.
 		a.handle = &handle{}
```

Comment:
Due to rebase I guess, some methods now lack the guard 
For example, ListConsumerGroups,  DescribeConsumerGroups and so on